app/subscriber/src/features/search-page/components/advanced-search/AdvancedSearch.tsx

```diff
@@ -341,40 +372,45 @@ export const AdvancedSearch: React.FC<IAdvancedSearchProps> = ({ onSearch, setSe
               <ExpandableRow icon={<FaGears />} title="Advanced Options:" hasValues={false}>
                 <Row alignItems="center" justifyContent="space-between">
                   Default operator:
-                  <ToggleGroup
-                    defaultSelected={search.defaultOperator ?? 'and'}
-                    options={[
-                      {
-                        id: 'and',
-                        label: 'AND',
-                        onClick: () => storeSearchFilter({ ...search, defaultOperator: 'and' }),
-                      },
-                      {
-                        id: 'or',
-                        label: 'OR',
-                        onClick: () => storeSearchFilter({ ...search, defaultOperator: 'or' }),
-                      },
-                    ]}
-                  />
+                  <FormikForm initialValues={{}} onSubmit={() => {}}>
```

Comment:
Why introduce the additional formik form when it's not being used anyway?

Reply:
without this additional formik, the ui throws exception complaining about reading undefined values.

Reply:
Then all this is doing is hiding the issue.  The form is throwing undefined because the initial value should never be undefined.

Reply:
If you're hiding the error this way it means your types or implementation has issues.  Typescript is giving you errors because it's tell you something is wrong.  If you ignore it by essentially using `any` then there is no point in using Typescript.

Reply:
updated. please review again. thanks!